sort.js

```diff
@@ -0,0 +1,228 @@
+  'Liberia': '🇱🇷',
+  'Libya': '🇱🇾',
+  'Liechtenstein': '🇱🇮',
+  'Lithuania': '🇱🇹',
+  'Luxembourg': '🇱🇺',
+  'Macedonia': '🇲🇰',
+  'Madagascar': '🇲🇬',
+  'Malawi': '🇲🇼',
+  'Malaysia': '🇲🇾',
+  'Maldives': '🇲🇻',
+  'Mali': '🇲🇱',
+  'Malta': '🇲🇹',
+  'Marshall Islands': '🇲🇭',
+  'Mauritania': '🇲🇷',
+  'Mauritius': '🇲🇺',
+  'Mexico': '🇲🇽',
+  'Micronesia': '🇫🇲',
+  'Moldova': '🇲🇩',
+  'Monaco': '🇲🇨',
+  'Mongolia': '🇲🇳',
+  'Montenegro': '🇲🇪',
+  'Morocco': '🇲🇦',
+  'Mozambique': '🇲🇿',
+  'Myanmar': '🇲🇲',
+  'Namibia': '🇳🇦',
+  'Nauru': '🇳🇷',
+  'Nepal': '🇳🇵',
+  'Netherlands': '🇳🇱',
+  'New Zealand': '🇳🇿',
+  'Nicaragua': '🇳🇮',
+  'Niger': '🇳🇪',
+  'Nigeria': '🇳🇬',
+  'Niue': '🇳🇺',
+  'North Korea': '🇰🇵',
+  'Norway': '🇳🇴',
+  'Oman': '🇴🇲',
+  'Pakistan': '🇵🇰',
+  'Palau': '🇵🇼',
+  'Panama': '🇵🇦',
+  'Papua New Guinea': '🇵🇬',
+  'Paraguay': '🇵🇾',
+  'China': '🇨🇳',
+  'Peru': '🇵🇪',
+  'Philippines': '🇵🇭',
+  'Poland': '🇵🇱',
+  'Portugal': '🇵🇹',
+  'Qatar': '🇶🇦',
+  'Republic of Congo': '🇨🇬',
+  'Romania': '🇷🇴',
+  'Russia': '🇷🇺',
+  'Rwanda': '🇷🇼',
+  'Saint Kitts and Nevis': '🇰🇳',
+  'Saint Lucia': '🇱🇨',
+  'Saint Vincent and Grenadines': '🇻🇨',
+  'Samoa': '🇼🇸',
+  'San Marino': '🇸🇲',
+  'Sao Tome and Principe': '🇸🇹',
+  'Saudi Arabia': '🇸🇦',
+  'Senegal': '🇸🇳',
+  'Serbia': '🇷🇸',
+  'Seychelles': '🇸🇨',
+  'Sierra Leone': '🇸🇱',
+  'Singapore': '🇸🇬',
+  'Slovakia': '🇸🇰',
+  'Slovenia': '🇸🇮',
+  'Solomon Islands': '🇸🇧',
+  'Somalia': '🇸🇴',
+  'South Africa': '🇿🇦',
+  'South Korea': '🇰🇷',
+  'South Sudan': '🇸🇸',
+  'Spain': '🇪🇸',
+  'Sri Lanka': '🇱🇰',
+  'Sudan': '🇸🇩',
+  'Suriname': '🇸🇷',
+  'Swaziland': '🇸🇿',
+  'Sweden': '🇸🇪',
+  'Switzerland': '🇨🇭',
+  'Syria': '🇸🇾',
+  'Tajikistan': '🇹🇯',
+  'Tanzania': '🇹🇿',
+  'Thailand': '🇹🇭',
+  'Togo': '🇹🇬',
+  'Tonga': '🇹🇴',
+  'Trinidad and Tobago': '🇹🇹',
+  'Tunisia': '🇹🇳',
+  'Turkey': '🇹🇷',
+  'Turkmenistan': '🇹🇲',
+  'Tuvalu': '🇹🇻',
+  'Uganda': '🇺🇬',
+  'Ukraine': '🇺🇦',
+  'United Arab Emirates': '🇦🇪',
+  'United Kingdom': '🇬🇧',
+  'United States': '🇺🇸',
+  'Uruguay': '🇺🇾',
+  'Uzbekistan': '🇺🇿',
+  'Vanuatu': '🇻🇺',
+  'Vatican City': '🇻🇦',
+  'Venezuela': '🇻🇪',
+  'Vietnam': '🇻🇳',
+  'Western Sahara': '🇪🇭',
+  'Yemen': '🇾🇪',
+  'Zambia': '🇿🇲',
+  'Zimbabwe': '🇿🇼'
+};
+(async () => {
+  const markdown = list
+    .map(({ name, country, github }) => {
+      let text = '- '
+      if (name) {
+        text += name
+      }
+      if (country && country !== '?') {
+        text += ` from ${country} `
+        if (emojiTable[country]) {
+          text += emojiTable[country]
+        }
+      }
+      if (github) {
+        text += ` ([${github}](https://github.com/${github}))`
+      }
+      return text
+    })
+    .sort()
+    .join('\n')
+  let readme = String(await readFile('README.md'))
+  readme = readme.replace(/<!-- SUPPORTERS_START -->((.|\n)*)<!-- SUPPORTERS_END -->/,
+    `<!-- SUPPORTERS_START -->\n${markdown}\n<!-- SUPPORTERS_END -->`)
+  await writeFile('README.md', readme)
+})()
```

Comment:
```suggestion
})().catch(console.error)
```

Just in case something goes really bad, don't crash the process.